docs/build/using-local-fluence-network.md

```diff
@@ -0,0 +1,68 @@
+# Using local Fluence Network
+
+The simplest way to test and debug your code is to run it on a local Fluence Network using docker.
+
+## Prerequisites
+
+- Install [Fluence CLI](https://github.com/fluencelabs/fluence-cli)
+
+- Ths simplest way to use docker is to install [Docker Desktop](https://docs.docker.com/desktop/) (We recommend using [this guide](https://docs.docker.com/desktop/) to install docker from official documentation for you OS)
+
+
+## Run local Fluence Network
+
+Right now CLI works with 4 different environments `kras` (production environment), `testnet`, `stage` (bleeding edge) and `local`. The environment you select controls which chain, IPFS, Nox addresses, etc. will be used when you running CLI commands 
+
+If you already have a Fluence Project `cd` into it and set local environment as default like this:
+```sh
+fluence default env local
+``` 
+Or you can init a new Fluence Project and choose a local environment for it like this
+```sh
+fluence init --env local
+```
+
+Make sure docker daemon is running (e.g. by just running Docker Desktop application) and then run:
+```sh
+fluence local up
+```
+
+After that you will see a `docker-compose.yaml` file generated in your project's `.fluence` dir. Then CLI will run `docker compose up -d` for you and it will also register a default provider and offer on your local Fluence Network. Currently CLI has a couple of convenience commands that wrap common docker commands (run `fluence local --help` to learn more), but we encourage you to use docker itself as you please
+
+The default local Fluence Network at this moment has:
+- 3 Nox containers
+- IPFS container
+- Couple of containers required for operations related to blockchain
+
+If you want to change amount of Noxes or change default provider and offer you can update `provider.yaml` file and run `fluence local init` to generate a new `.fluence/docker-compose.yaml`. Required `secrets` and `configs` in `.fluence` will be also generated for you automatically
+
+You can also change `.fluence/docker-compose.yaml` directly any way you want
```

Comment:
```suggestion
You can also change `.fluence/docker-compose.yaml` directly any way you want, but be aware that it will be regenerated on `fluence local init`, and your changes might be lost.
```